libkbfs/disk_block_cache_helper.go

```diff
@@ -26,6 +26,8 @@ type diskBlockCacheMetadata struct {
 	BlockSize uint32
 	// whether the block has triggered prefetches
 	HasPrefetched bool
+	// whether the block's triggered prefetches are complete
+	DonePrefetch bool
```

Comment:
Can we update this name?  I know we can't update `HasPrefetched` since it's serialized, but it would be nice if this matched the name in the interface.